### ISSUE TITLE
add percentage of the screen width to pictures

### ISSUE THREAD

**Issue by insaneinthemind** [insaneinthemind/insaneinthemind.github.io#3]

**Comment by jhawlc** [comment]
Your images should resize if the pages resize
There should be alt text on each image to describe it.

Example:

`<img src ="images/arms.jpg">`

Should be

`<img src ="images/arms.jpg" alt="Wrist Tech" width=20% height=10%>`
